authors/apps/profiles/tests.py

```diff
@@ -27,3 +27,12 @@ def test_edit_my_profile_succeeds(self):
             HTTP_AUTHORIZATION='Bearer ' + self.test_user_token
             )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_authenticated_user_view_author_profiles_succeeds(self):
```

Comment:
comments NEEDED here @armstrongsouljah

Reply:
Thanks for spotting that out. Let me resolve that.

Reply:
@MuhweziDeo I added the recommended docs in the test files.